lib/src/interface/common.dart

```diff
@@ -85,7 +85,26 @@ String? getExecutablePath(
     candidates = _getCandidatePaths(command, searchPath, extensions, context);
   }
   for (String path in candidates) {
```

Comment:
I don't disagree with this change, but in context it seems like this is going to cause errors that the flutter tool current surfaces as permission errors to instead be surfaced as an argument error. This is going to hide problems for our users.